cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -964,6 +964,15 @@ if(builtin_xrootd)
   set(XROOTD_CFLAGS "-DROOTXRDVERS=${XROOTD_VERSIONNUM}")
   install(DIRECTORY ${XROOTD_ROOTDIR}/${XROOTD_LIBDIR}/ DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libraries FILES_MATCHING PATTERN "libXrd*")
   install(DIRECTORY ${XROOTD_ROOTDIR}/include/xrootd/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)
+  if(NOT WINDOWS)
```

Comment:
@bellenot do we need this here? I guess `builtin_xrootd` is always going to be OFF on windows?)

Reply:
No, I think it's useless. And `builtin_xrootd` is OFF by default (even if one can try to set it to ON)